app/policies/account_warning_policy.rb

```diff
@@ -6,11 +6,15 @@ def show?
   end
 
   def appeal?
-    target? && record.created_at >= Appeal::MAX_STRIKE_AGE.ago
+    target? && eligible_for_appeal?
   end
 
   private
 
+  def eligible_for_appeal?
+    record.created_at >= Appeal::MAX_STRIKE_AGE.ago
```

Comment:
Unrelated to this PR, I think this constant is used more often from the perspective of a strike (`AccountWarning`) than from `Appeal`  ... could see moving this to `AccountWarning::APPEALABLE_DURATION` (or whatever) if that's indeed the case.